index.html

```diff
@@ -8,12 +8,41 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/About.css">
 
 </head>
 
 <body>
-    <div class="container">
```

Comment:
You need to add a    div class=wrapper and wrap your whole html.  if you don't your page is expanded and it doesn't look like the design file.